components/newsletter/newsletter.html

```diff
@@ -0,0 +1,56 @@
+<aside class="mzp-c-newsletter">
+  <div class="mzp-c-newsletter-image">
+    <img src="{{ '/img/newsletter-image.png' | path }}" width="346" height="346" alt="">
+  </div>
+
+  <form id="newsletter-form" class="mzp-c-newsletter-form" name="newsletter-form" action="#" method="post">
```

Comment:
Gg with the addition of the  variants, I think this is a nice way of showing things!

However, still getting something like this in the initial form submit.
<img width="463" alt="Screen Shot 2022-06-22 at 10 15 05 AM" src="https://user-images.githubusercontent.com/42309026/174916931-8907159e-d6a8-462a-9806-fb06069ebccc.png">
Nothing blocking, but we could remove `method="post"` and change it to `method=""` to prevent it from trying to submit and we could leave a comment in the notes saying there is no `method` attribute just for demonstration purposes. 

Again, nothing blocking, just less error-y would be cool.

Reply:
oh good idea